### PR TITLE
SP: Add markup for login page

### PR DESCRIPTION
**Pull request by AnnaRokalo**

### CODE REVIEW

File: src/pages/auth/LoginForm.tsx [hunk 1]
```diff
@@ -67,8 +67,9 @@ export const LoginForm = ({
       />
 
       <Spacer size={32} />
-
-      <Button type="submit" loading={isLoading}>
+      <Text>Remember me</Text>
+      <Spacer size={32} />
+      <Button type="submit" loading={isLoading} css={{width: '100%', backgroundColor: '$system-black', color: '$system-white'}}>
```
Comment (VladKarchenia) [inline src/pages/auth/LoginForm.tsx]: 1. Leave this button yellow or add another variant (black button) into `<Button>` component
2. There is full prop to make it full width

File: src/pages/auth/LoginForm.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import { useState } from "react"
 import { Controller, useFormContext } from "react-hook-form"
-import { Button, FormInput, Spacer } from "@/shared/components"
-import { IconTick } from "@/shared/icons"
+import { Button, FormInput, Spacer, Text } from "@/shared/components"
 
 import { LoginInput } from "./LoginFormContainer"
 import { SShowPasswordButton } from "./LoginForm.styles"
+import { IconClarityEye } from "@/shared/icons/IconClarityEye"
```
Comment (VladKarchenia) [inline src/pages/auth/LoginForm.tsx]: import @/shared/icons

File: src/pages/auth/LoginForm.tsx [hunk 3]
```diff
@@ -67,8 +67,9 @@ export const LoginForm = ({
       />
 
       <Spacer size={32} />
-
-      <Button type="submit" loading={isLoading}>
+      <Text>Remember me</Text>
```
Comment (VladKarchenia) [inline src/pages/auth/LoginForm.tsx]: use `<Copy>` instead of `<Text>`

File: src/shared/icons/IconClarityEye/IconClarityEye.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { withIcon } from "../Icon"
+
+export const IconClarityEye = withIcon(
```
Comment (VladKarchenia) [inline src/shared/icons/IconClarityEye/IconClarityEye.tsx:3]: use app to compose svg

File: src/pages/auth/Login.tsx [hunk 5]
```diff
@@ -1,29 +1,43 @@
...
+            </Box>
+            <Box
+              css={{
+                display: "flex",
+                justifyContent: "center",
+                flexDirection: "column",
+                flex: '1 1 100%',
+              }}
+              >
+                <Title css={{marginBottom: '$24'}}>Welcome!</Title>
```
Comment (VladKarchenia) [inline src/pages/auth/Login.tsx]: when we use `<Title>` we need to specify "as" prop if it's not `<h1>` tag

File: src/pages/auth/Login.tsx [hunk 6]
```diff
@@ -1,29 +1,43 @@
...
+            backgroundColor: "$neutrals-1",
+            padding: '$32 $24',
+            display: "flex",
+            flexFlow: 'column nowrap',
+            justifyContent: 'center'
+          }} >
+            <Box css={{flex: '0 0 auto',}}>
+              <Logo />
+            </Box>
+            <Box
```
Comment (VladKarchenia) [inline src/pages/auth/Login.tsx]: we can use `<Flex>` here with props like "direction", "justify" etc.

**Comment by VladKarchenia** [comment]
1. When adding a new icon, you need to add another export to then just import the icon from @/icons
2. https://jakearchibald.github.io/svgomg/ we can use this tool, to compress svg icons, after that icon code will be
```
export const IconClarityEye = withIcon(
  '<path d="M18.678 9.739c-1.872-3.461-5.156-5.556-8.79-5.556-3.632 0-6.921 2.095-8.777 5.556L.956 10l.144.267c1.872 3.46 5.156 5.555 8.789 5.555s6.922-2.066 8.789-5.555l.144-.267-.144-.261Zm-8.79 4.944c-3.127 0-6-1.744-7.666-4.683C3.89 7.061 6.762 5.317 9.89 5.317c3.128 0 5.967 1.75 7.661 4.683-1.694 2.939-4.539 4.683-7.661 4.683Z" fill="#000" fill-opacity=".5"/><path d="M10.05 6.206a3.811 3.811 0 1 0 .056 7.622 3.811 3.811 0 0 0-.056-7.622Zm0 6.51a2.7 2.7 0 1 1 .056-5.399 2.7 2.7 0 0 1-.056 5.4Z" fill="#000" fill-opacity=".5"/>',
)
```

File: src/pages/auth/Login.tsx [hunk 7]
```diff
@@ -1,29 +1,43 @@
...
+          <GridItem css={{
+            backgroundColor: "$neutrals-1",
+            padding: '$32 $24',
+            display: "flex",
+            flexFlow: 'column nowrap',
+            justifyContent: 'center'
+          }} >
+            <Box css={{flex: '0 0 auto',}}>
+              <Logo />
+            </Box>
```
Comment (VladKarchenia) [inline src/pages/auth/Login.tsx:26]: pls add gap between logo and form title for mobile view

File: src/pages/auth/LoginForm.tsx (hunk 1 above)
Comment (AnnaRokalo) [reply src/pages/auth/LoginForm.tsx]: Fixed

File: src/pages/auth/Login.tsx (hunk 7 above)
Comment (AnnaRokalo) [reply src/pages/auth/Login.tsx:26]: fixed

File: src/pages/auth/LoginForm.tsx (hunk 3 above)
Comment (AnnaRokalo) [reply src/pages/auth/LoginForm.tsx]: fixed

File: src/pages/auth/Login.tsx (hunk 6 above)
Comment (AnnaRokalo) [reply src/pages/auth/Login.tsx]: Here I need flex property that set flex-shrink, flex-growth, flex-basis, we don't have them in <Flex> component. Should I extand the component?

**Pull request merged**